package.json

```diff
@@ -4,16 +4,15 @@
   "description": "A Social platform for the creative at heart",
   "main": "index.js",
   "scripts": {
-    "coverage": "npm run db-migrate && mocha --reporter mochawesome --timeout=20000 --require babel-register --exit",
+    "coverage": "nyc report --reporter=text-lcov | coveralls",
     "db-migrate": "sequelize db:migrate",
     "db:undo-migration": "sequelize db:migrate:undo:all",
     "lint": "eslint routes models config test index.js",
-    "test": "npm run db:undo-migration && npm run db-migrate && mocha test/index.js --timeout=20000 --require babel-register --exit",
-    "start": "sequelize db:migrate && node dist/index",
+    "start": "sequelize db:migrate && npm run build && node dist/ndex.js",
```

Comment:
@kleva-j there's still an issue here. You're not referencing the same file. You should take time to go through all you've done to avoid anything you might have overlooked